src/main/io/ledstrip.c

```diff
@@ -794,57 +933,51 @@ static void applyLedAnimationLayer(bool updateNow, uint32_t *timer)
 #endif
 
 typedef enum {
-    timIndicator,
     timBlink,
-    timWarning,
+    timLarson,
+    timBattery,
+    timRssi,
 #ifdef GPS
     timGps,
 #endif
-    timRotation,
+    timWarning,
+    timIndicator,
 #ifdef USE_LED_ANIMATION
     timAnimation,
 #endif
+    timRing,
     timTimerCount
 } timId_e;
 
 static uint32_t timerVal[timTimerCount];
 
+
 // function to apply layer.
 // function must replan self using timer pointer
 // when updateNow is true (timer triggered), state must be updated first,
 //  before calculating led state. Otherwise update started by different trigger
 //  may modify LED state.
 typedef void applyLayerFn_timed(bool updateNow, uint32_t* timer);
-typedef void applyLayerFn(void);
 
-static const struct {
-    int8_t timId;                          // timer id for update, -1 if none
-    union {
-        applyLayerFn *apply;               // function to apply layer unconditionally
-        applyLayerFn_timed *applyTimed;    // apply with timer
-    } f;
-} layerTable[] = {
-    // LAYER 1
-    { -1,             .f.apply      = &applyLedModeLayer },
-    { -1,             .f.apply      = &applyLedHueLayer },
-    // LAYER 2
-    {timWarning,      .f.applyTimed = &applyLedWarningLayer},
+
+static applyLayerFn_timed* layerTable[] = {
+    &applyLedBlinkLayer,
```

Comment:
use designated initializers here to prevent  posible errors:
`[timBlink] = &applyLedBlinkLayer,`